mbedtls/src/pkcs12/mod.rs

```diff
@@ -356,6 +356,7 @@ impl BERDecodable for EncryptedData {
 
 // EncryptedContentInfo from PKCS7 see RFC 2315 section 10.1
 #[derive(Debug, Clone)]
+#[allow(dead_code)]
```

Comment:
I am not fond of these changes.

If this needed for feature compilation? In that case the struct should be behind a feature. 
Is it needed just to get rid of the compiler warning, but later on the field is reintroduced and used?

In both cases, don't fix now so we get this unblocked, but could we open an issue to remove `allow(dead_code)` after we finish with all of this?

Reply:
Sure. This is intoduced when `cargo test --features pkcs12` these two struct become dead code and thus blocked by `#![deny(warnings)]` in `src/lib.rs`

Reply:
Will create a issue for this after merging the #225 & #127

Reply:
We could leave these `#[allow(dead_code)]` in #225 and #127
And I could try to remove them in #224